brewtils/plugin.py

```diff
@@ -352,8 +349,33 @@ def _initialize_logging(self):
             self._logger.debug("Skipping logging init: custom logger detected")
             return
 
-        bg_log_config = self._ez_client.get_logging_config(self._system.name)
-        logging.config.dictConfig(convert_logging_config(bg_log_config))
+        try:
+            log_config = self._ez_client.get_logging_config()
+        except Exception as ex:
+            self._logger.warning(
+                "Unable to retrieve logging configuration from Beergarden, the default "
+                "configuration will be used instead. Caused by: {0}".format(ex)
+            )
+            return
+
+        try:
+            configure_logging(
+                log_config,
+                namespace=self._system.namespace,
+                system_name=self._system.name,
+                system_version=self._system.version,
+                instance_name=self._config.instance_name,
+            )
```

Comment:
Since we now have the functionality, what is preventing the user from creating a logging-config.yaml and passing that instead of creating a custom logger? 

This would allow them to have "special" loggers for specific plugins. So maybe 9 out of the 10 plugins don't really matter whats in the logs, but the 10th one is super important and they want a longer rotation for holding the logs before delete?

Should we add a logging config input parameter then run in this priority:
1. User provided initialized custom logger
1. User provided logging config
1. Server provided logging config
1. No Logging Setup

Reply:
Yeah, that seems like a good idea. It's going in a new ticket though :smile: